templates/result.html

```diff
@@ -1,67 +1,66 @@
-            <!-- Image Persentase -->
-            <div class="p-5 flex flex-col items-center justify-center gap-3">
-                <img src="/static/images/Persen.png" alt="Image 4" class="w-[250px]">
-            </div>
+    <section class="max-w-6xl mx-auto p-8">
+    <!-- Grid Upload Boxes -->
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+        <!-- Image Persentase -->
+        <div class="p-5 flex flex-col items-center justify-center gap-3">
+            <img src="/static/images/Persen.png" alt="Image 4" class="w-[250px]" />
+        </div>
+        <!-- Result Plagiarism -->
+        <div class="flex flex-col justify-between">
+            <p class="text-gray-700 mb-8 text-lg">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+                minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                aliquip ex ea commodo consequat. Duis aute irure dolor in
+                reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+                pariatur. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                laboris nisi ut aliquip ex ea commodo consequat.
+            </p>
 
-            <!-- Result Plagiarism -->
-            <div class="flex flex-col justify-between">
-                <p class="text-gray-700 mb-8 text-lg">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore 
-                    magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo 
-                    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-                    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo 
-                    consequat. 
-                </p>
-                
-                <!-- Download Button -->
-                <div class="flex justify-center">
-                    <button class="bg-gradient-to-r from-deep-blue to-light-blue text-white font-bold py-3 px-6 rounded-lg hover:from-teal-500 hover:to-teal-700 transition-all duration-300">
-                        Download PDF
-                    </button>
-                </div>
+            <!-- Download Button -->
+            <div class="flex justify-center">
+                <button
+                    class="bg-gradient-to-r from-deep-blue to-light-blue text-white font-bold py-3 px-6 rounded-lg hover:from-teal-500 hover:to-teal-700 transition-all duration-300">
+                    Download PDF
+                </button>
             </div>
         </div>
-    </section>
-
-
-    <script src="/static/js/script.js"></script>
+    </div>
+</section>
     <script src="https://cdn.jsdelivr.net/npm/flowbite@2.5.2/dist/flowbite.min.js"></script>
```

Comment:
please put all script tag in the head element for easier tracking, thanks!

Reply:
Okay, sir!

Reply:
okay done